mod/groups/views/default/groups/edit/access.php

```diff
@@ -8,128 +8,92 @@
-			"value" => $visibility,
-			"options_values" => $visibility_options,
-			'entity' => $entity,
-			'entity_type' => 'group',
-			'entity_subtype' => '',
-		));
-		?>
-	</div>
-<?php endif; ?>
-
-<?php
+	$visibility_options = [
+		ACCESS_PRIVATE => elgg_echo('groups:access:group'),
+		ACCESS_LOGGED_IN => elgg_echo('LOGGED_IN'),
+		ACCESS_PUBLIC => elgg_echo('PUBLIC'),
+	];
+	if (elgg_get_config('walled_garden')) {
+		unset($visibility_options[ACCESS_PUBLIC]);
+	}
 
-$access_mode_params = array(
-	"name" => "content_access_mode",
-	"id" => "groups-content-access-mode",
-	"value" => $content_access_mode,
-	"options_values" => array(
-		ElggGroup::CONTENT_ACCESS_MODE_UNRESTRICTED => elgg_echo("groups:content_access_mode:unrestricted"),
-		ElggGroup::CONTENT_ACCESS_MODE_MEMBERS_ONLY => elgg_echo("groups:content_access_mode:membersonly"),
-	)
-);
+	echo elgg_view_input('access', [
+		'name' => 'vis',
+		'id' => 'groups-vis',
+		'value' => $visibility,
+		'options_values' => $visibility_options,
+		'entity' => $entity,
+		'entity_type' => 'group',
+		'entity_subtype' => '',
+		'label' => elgg_echo('groups:visibility'),
+	]);
+}
 
-if ($entity) {
+echo elgg_view_input('select', [
+	'name' => 'content_access_mode',
+	'id' => 'groups-content-access-mode',
+	'value' => $content_access_mode,
+	'options_values' => [
+		ElggGroup::CONTENT_ACCESS_MODE_UNRESTRICTED => elgg_echo('groups:content_access_mode:unrestricted'),
+		ElggGroup::CONTENT_ACCESS_MODE_MEMBERS_ONLY => elgg_echo('groups:content_access_mode:membersonly'),
+	],
+	'label' => elgg_echo('groups:content_access_mode'),
+	// Warn the user that changing the content access mode to more
+	// restrictive will not affect the existing group content
+	'help' => $entity && $entity->getContentAccessMode() == ElggGroup::CONTENT_ACCESS_MODE_UNRESTRICTED ? elgg_echo('groups:content_access_mode:warning') : '',
 	// Disable content_access_mode field for hidden groups because the setting
 	// will be forced to members_only regardless of the entered value
-	if ($entity->access_id === $entity->group_acl) {
-		$access_mode_params['disabled'] = 'disabled';
-	}
-}
-?>
-<div>
-	<label for="groups-content-access-mode"><?php echo elgg_echo("groups:content_access_mode"); ?></label><br />
-	<?php
-		echo elgg_view("input/select", $access_mode_params);
-
-		if ($entity && $entity->getContentAccessMode() == ElggGroup::CONTENT_ACCESS_MODE_UNRESTRICTED) {
-			// Warn the user that changing the content access mode to more
-			// restrictive will not affect the existing group content
-			$access_mode_warning = elgg_echo("groups:content_access_mode:warning");
-			echo "<span class='elgg-text-help'>$access_mode_warning</span>";
-		}
-	?>
-</div>
-
-<?php
+	'disabled' => $entity && $entity->access_id === $entity->group_acl,
+]);
 
 if ($entity && ($owner_guid == elgg_get_logged_in_user_guid() || elgg_is_admin_logged_in())) {
-	$members = array();
+	$members = [];
 
 	$dbprefix = elgg_get_config('dbprefix');
-	$options = array(
```

Comment:
In the future please resist the urge to change stuff like this. It doesn't add readability, it adds noise to the PR and git history.